src/app/grants/education/whoCanApply.tsx

```diff
@@ -14,7 +14,7 @@ export default function WhoCanApply({
   const details = [
     "Bitcoin education programs focused on training and upskilling Bitcoin developers.",
     "Studies, white papers, and data-driven analysis exploring Bitcoin's impact, technical advancements, and adoption trends in the Global South.",
```

Comment:
Will need to take out this, too, like below
```suggestion
    // "Studies, white papers, and data-driven analysis exploring Bitcoin's impact, technical advancements, and adoption trends in the Global South.",
```